docs/install/database.md

```diff
@@ -0,0 +1,88 @@
+## Recommendation
+
+For production deployments, we recommend using an external [PostgreSQL](https://www.postgresql.org/) database (version 13 or higher).
+
+## Basic configuration
+
+Before starting the Coder server, prepare the database server. Create a role and a database.
+Remember that the role must have access to the created database.
+
+With `psql`:
+
+```sql
+CREATE ROLE coder LOGIN SUPERUSER PASSWORD 'secret42';
+```
+
+With `psql -U coder`:
+
+```sql
+CREATE DATABASE coder;
+```
+
+Coder configuration is defined via [environment variables](../admin/configure.md).
+The database client requires the connection string provided via the `CODER_PG_CONNECTION_URL` variable.
+
+```sh
+export CODER_PG_CONNECTION_URL="postgres://coder@localhost/coder?password=secret42&sslmode=disable"
```

Comment:
I've only seen the URL formatted as `postgres://coder:secret42@localhost/coder?sslmode=disable` before, does the password query param work?

Reply:
Yeah, these are interchangeable, but I can adjust to the standard pattern `user:password`.

Reply:
We seem to actually be using a mixture of both. It might be a good idea for us to standardize behind one usage (see `rg postgres://` in repo). But that would be something for a separate PR.